app/src/main/java/com/kickstarter/viewmodels/projectpage/AddOnsViewModel.kt

```diff
@@ -191,8 +191,6 @@ class AddOnsViewModel(val environment: Environment) : ViewModel() {
             emitCurrentState()
         }
 
-        if (shippingSelectorIsGone) getAddOns(noShippingRule = true)
-
```

Comment:
Some logic changed in this viewmodel that makes this unnecessary now, was getting add-ons twice in digital reward cases